src/models/userModel.js

```diff
@@ -0,0 +1,30 @@
+import { model, Schema } from 'mongoose'
+
+const schema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  surname: {
+    type: String,
+    required: true,
+  },
```

Comment:
I think we don't need this attributes right know. Delete them or at least don't required them. I think we should just require email and password. Maybe after we can integrate with google and facebook to register the user, in that case we add the name and surname to fill our database.